.gitlab/deploy_containers/deploy_containers_a7.yml

```diff
@@ -95,12 +95,16 @@ deploy_containers-a7_internal-rc:
     VERSION: 7-rc
 
 
-deploy_containers-ot:
+.deploy_containers-versioned-ot:
   extends: .deploy_containers-a7-base-ot
   before_script:
+    - source /root/.bashrc
```

Comment:
Is this line still needed ?
I thought all `source /root/.bashrc` lines could be removed from everywhere in `.gitlab-ci.yml` because it was now done in the `ENTRYPOINT` of the docker build image.
* https://github.com/DataDog/datadog-agent/pull/29007
* https://github.com/DataDog/datadog-agent/pull/29237

Reply:
Actually, that line is in fact what fixes it. I just tested it with a manually triggered pipeline on this branch. We should probably merge this, fix the CI images, and then maybe revert the fix. We could go straight to fixing the CI images, but I'm not sure I want to wait for that to happen 😅